ui/package.json

```diff
@@ -45,9 +45,9 @@
         "@types/mocha": "^2.2.32",
```

Comment:
FYI you're gonna have to bump the version of the ui package again because I already published a 0.16.5 version this morning